tests/scripts/task_riscv_microtvm.sh

```diff
@@ -22,6 +22,13 @@ source tests/scripts/setup-pytest-env.sh
 
 make cython3
 
-# NOTE: this exists to ensure some tests run on RISC-V image. Without it, Jenkins reports a configuration error.
-# This line can be removed when RISC-V tests are added.
-run_pytest ctypes riscv-platform-minimal-test-0 tests/python/all-platform-minimal-test
+#riscv gcc
+export PATH=/opt/csi-nn2/tools/gcc-toolchain/bin:$PATH
+#riscv qemu
+export PATH=/opt/csi-nn2/tools/qemu/bin:$PATH
+
+# run qemu tvm_rpc
+nohup qemu-riscv64 -cpu c906fdv -L /opt/csi-nn2/tools/gcc-toolchain/sysroot/ ./build-c906/tvm_rpc server --host=127.0.0.1 --port=9090 &
```

Comment:
what do you think about launching this from a pytest [fixture](https://docs.pytest.org/en/6.2.x/fixture.html)?

Reply:
a very good suggestion. pytest.fixture is very useful for test programs that need preprocessing and post-processing